applespi.c

```diff
@@ -1,3 +1,13 @@
+/*
+ * MacBook (Pro) SPI keyboard and touchpad driver
+ *
+ * Copyright (c) 2015-2017 Federico Lorenzi
```

Comment:
@roadrunner2: Given the work you put into that driver, you should add yourself there as author as well. You truly deserve it!

Reply:
Thanks, but it's fine. Federico did the original work, without which I wouldn't have had anything to contribute to.

Reply:
I definitely agree with @Dunedan here, you've done all the heavy lifting into getting the reverse engineered proof of concept I had into shape!